src/SimpleFeatureToggle/FileFeatureLoadingStrategy.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System.Diagnostics;
+
+namespace SimpleFeatureToggle
+{
+    public class FileFeatureLoadingStrategy : IFeatureLoadingStrategy
+    {
+        private readonly IFileReader _fileReader;
+        private string _fileName;
+
+        public FileFeatureLoadingStrategy(string fileName = null, IFileReader fileReader = null)
```

Comment:
Yeah, I agree. I think when I did this the intention was to try and keep the old API available, but then I stopped short of that because it sucked. I'll rip these bits out. 👍